docs/painless/painless-contexts/painless-field-context.asciidoc

```diff
@@ -26,3 +26,64 @@ a customized value for each document in the results of a query.
 *API*
 
 The standard <<painless-api-reference, Painless API>> is available.
+
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+We will use two scripts to create two additional fields for each search hit.
+These script fields will output computed custom information.
+
+The first script calculates the day of the week for the field `datetime`:
+
+[source,Painless]
+----
+doc['datetime'].value.getDayOfWeek();                               <1>
+----
+
+<1> Accesses a doc value for the `datetime` field, and uses the function
+    `getDayOfWeek` to calculate the day of the week for this date.
+
+
+
+The second script calculates the number of actors. Actors' names are stored
+as an array in the `actors` field:
+
```

Comment:
I'd say "as a text array" -- then you don't have to reitrate that `actors` is a text field in the callout. 

I'd  also change the :  to a . to be consistent with the punctuation above. (And the second sentence adds info, rather than really leading into the example.)